src/routes.json

```diff
@@ -181,6 +181,13 @@
         "title": "Jobs"
     },
     {
+        "name": "jobs-moderator",
+        "pattern": "^jobs/moderator/?$",
```

Comment:
This needs an extra `/` before `jobs`: I think it should be `^/jobs/moderator/?$`